docs/api/app.md

```diff
@@ -147,18 +166,19 @@ A dispatch initializer that can create a [custom dispatch function](../architect
 ```js
 app({
   init: { message: "Hello, World!" },
-  view: (state) => h("p", {}, text(state.message)),
+  view: (state) => h("body", {}, h("p", {}, text(state.message))),
```

Comment:
Let's use `main` instead of `body`.

Reply:
with `node: document.querySelector('main')`